vm/native/native.cpp

```diff
@@ -1,4 +1,6 @@
 //#define GC_DEBUG
+//#define GC_MAXIMUM_HEAP_SIZE (128 * 1024 * 1024)
```

Comment:
I think while debugging one wants to play with this value, so I'd just leave it as it is now.